### DISCUSSION TITLE
Linking to a documentation I wrote for making JSON requests to Rodauth Routes

### DISCUSSION THREAD

**Discussion by Samuelodan** [jeremyevans/rodauth#373] [General]
Hi there.
First of, thank you so much for building and maintaining Rodauth. I convinced my team to use it in our greenfield Rails app and it's been a pleasant experience so far; even though I hadn't looked into Roda until recently.

Over the past two months though, I've worked with Janko, the author of rodauth-rails, to craft a documentation for Rodauth's routes (just the bare minimum for now), and we believe it'll come in handy for folks looking to use Rodauth in an API app.
I'd like to make a PR to add a link to it in the resources section of Rodauth's README, but please take a look at it [here](https://documenter.getpostman.com/view/26686011/2s9YC7SWn9) to see if it's okay.

Thanks.

**Comment by jeremyevans** [comment]
I don't think I want to link to an external document in that section of the README.  However, having an "External Guides" section in the documentation page of the website, and linking to it from there, seems fine.

**Reply by Samuelodan** [reply]
Oh, my bad. I just realized that there's only first party resources in that section.
Alright. I'll make a PR to create the "External Guides" section on the site and link to. Thank you so much.
